### PR TITLE
Support new libmozevent features.

### PR DESCRIPTION
**Pull request by La0**

### CODE REVIEW

File: events/code_review_events/workflow.py [hunk 1]
```diff
@@ -277,24 +273,26 @@ def __init__(self, cache_root):
...
+                # autoland ingestion
+                exchanges[QUEUE_PULSE_AUTOLAND] = [
                     (PULSE_TASK_GROUP_RESOLVED, ["#.gecko-level-3.#"])
                 ]
             if publish:
                 # unit test failures
-                exchanges += [(PULSE_TASK_COMPLETED, ["*.*.gecko-level-3._"])]
+                exchanges[QUEUE_PULSE_TRY_TASK_END] = [
+                    (PULSE_TASK_COMPLETED, ["*.*.gecko-level-3._"])
+                ]
```
Comment (marco-c) [inline events/code_review_events/workflow.py]: I think this was an effect of some bad rebase, we can remove the lines from `if publish:` to here.

File: events/code_review_events/workflow.py [hunk 2]
```diff
@@ -277,24 +273,26 @@ def __init__(self, cache_root):
             self.webserver.register(self.bus)
 
             # Create pulse listener
-            exchanges = []
+            exchanges = collections.defaultdict(list)
```
Comment (marco-c) [inline events/code_review_events/workflow.py]: If you remove the few lines from the bad merge below, you can probably just use a normal dict here.

**Comment by marco-c** [comment]
Need to stop using libmozevent.taskcluster_config in bugbug_utils too.

**Comment by La0** [comment]
I already updated `bugbug_utils`

**Comment by marco-c** [comment]
> I already updated bugbug_utils

Ahah, not sure why I was not seeing it...

File: events/code_review_events/workflow.py (hunk 2 above)
Comment (La0) [reply events/code_review_events/workflow.py]: Done

File: events/code_review_events/workflow.py (hunk 1 above)
Comment (La0) [reply events/code_review_events/workflow.py]: Done

File: events/code_review_events/workflow.py [hunk 3]
```diff
@@ -277,24 +272,26 @@ def __init__(self, cache_root):
...
                     (PULSE_TASK_GROUP_RESOLVED, ["#.gecko-level-3.#"])
                 ]
-            if publish:
-                # unit test failures
-                exchanges += [(PULSE_TASK_COMPLETED, ["*.*.gecko-level-3._"])]
+
+            # unit test failures
+            exchanges[QUEUE_PULSE_TRY_TASK_END] = [
+                (PULSE_TASK_COMPLETED, ["*.*.gecko-level-3._"])
+            ]
```
Comment (marco-c) [inline events/code_review_events/workflow.py]: All these lines can be removed too (they were part of the bad rebase).

Comment (La0) [reply events/code_review_events/workflow.py]: done

**Pull request merged**